src/transformers/models/flaubert/modeling_tf_flaubert.py

```diff
@@ -596,15 +597,15 @@ def call(
         tensor = tensor * mask[..., tf.newaxis]
 
         # hidden_states and attentions cannot be None in graph mode.
-        hidden_states = ()
-        attentions = ()
+        hidden_states = () if inputs["output_hidden_states"] else None
+        attentions = () if inputs["output_attentions"] else None
 
         # transformer layers
         for i in range(self.n_layers):
             # LayerDrop
-            dropout_probability = tf.random.uniform([1], 0, 1)
+            dropout_probability = random.uniform(0, 1)
```

Comment:
Is Python's random better here?

Reply:
The TF compiler convert `random.uniform(0, 1)` into `tf.random.uniform([1], 0, 1)` so the final result is exactly the same at the end. I have updated this in order to be aligned with how it is done in TFBART.

Reply:
Why not leave it like it is, if you plan to update it later?

Reply:
Because it was raising an error with the condition below (cannot compare a tensor and not a tensor), so to do not spend too much time on this I prefered to do this and focus on this other problem later.